Change inferior to superior about limit/request in app-security-checklist (#48650)
kubernetes/website: content/en/docs/concepts/security/application-security-checklist.md

  when designing applications.
- [ ] Application configured with appropriate {{< glossary_tooltip text="QoS class" term_id="QoS-class" >}}
  through resource request and limits.

>   - [ ] Memory limit is set for the workloads with a limit equal or superior to the request.

Reviewer: ```suggestion
  - [ ] Memory limit is set for the workloads with a limit equal or greater than the request.
```
